config/migrations/20231109100512-new-minister-data.ttl

```diff
@@ -0,0 +1,148 @@
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/1ce11d20-8aa5-4b0d-b100-d6200125e20f> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/4341629b-40e6-4f37-aea3-62f0900a48dd> ;
+    ns1:rangorde 3 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "1ce11d20-8aa5-4b0d-b100-d6200125e20f" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/200195b1-d484-4e5f-91e0-d2a3cd44074e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0697> ;
+    ns1:rangorde 4 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "200195b1-d484-4e5f-91e0-d2a3cd44074e" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0703> .
+
+<http://themis.vlaanderen.be/id/mandatee/305a87c9-6806-42a0-81c9-f6a94c4e8464> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a06d2> ;
+    ns1:rangorde 7 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "305a87c9-6806-42a0-81c9-f6a94c4e8464" ;
+    dcterms:title "Vlaams minister van Mobiliteit en Openbare Werken" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/384269c5-542a-4c95-aec6-c6b250fca93e> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/74888cf5-6e19-4cf3-abf1-eeb9af999922> ;
+    ns1:rangorde 9 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "384269c5-542a-4c95-aec6-c6b250fca93e" ;
+    dcterms:title "Vlaams minister van Economie, Innovatie, Werk, Sociale Economie en Landbouw" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/658e57e6-4488-4aaf-bee7-1360ab362584> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907ee6670526694a0707> ;
+    ns1:rangorde 1 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "658e57e6-4488-4aaf-bee7-1360ab362584" ;
+    dcterms:title "Vlaams minister van Buitenlandse Zaken, Cultuur, Digitalisering en Facilitair Management" ;
+    org:holds <http://themis.vlaanderen.be/id/mandaat/5fed907ee6670526694a0704> .
+
+<http://themis.vlaanderen.be/id/mandatee/85e6f060-c8c5-4780-a399-e8b9b2d7d069> a ns1:Mandataris ;
+    ns1:isBestuurlijkeAliasVan <http://themis.vlaanderen.be/id/persoon/5fed907de6670526694a061b> ;
+    ns1:rangorde 2 ;
+    ns1:start "2023-11-08T00:00:00+01:00"^^xsd:dateTime ;
+    ns2:uuid "85e6f060-c8c5-4780-a399-e8b9b2d7d069" ;
+    dcterms:title "Vlaams minister van Welzijn, Volksgezondheid, Gezin en Zeevisserij" ;
```

Comment:
"Zeevisserij" is this new?
previous title: "Vlaams minister van Welzijn, Volksgezondheid en Gezin"

Reply:
mm :thinking: dat maakt me wat ongerust. Ik heb dat zeker niet zelf getypt, dus da's wat het script dacht dat de laatste titel was, maar ook op prod zie ik die nergens opgelijst :exploding_head: